german-quinonez/lesson-1/homework-2-Car/src/Car.java

```diff
@@ -0,0 +1,73 @@
+public class Car {
+
+    private double gas;
+    private double distance;
+    private double gasConsume;
+    private boolean isPowerOn;
+    private double capacity;
+
+
+    public Car() {
+        this.gas = 0.0;
+        this.distance = 0.0;
+        this.gasConsume = 0.0;
+        this.isPowerOn = false;
+        this.capacity = 0.0;
+    }
+
+    public void powerOn() {
+        this.isPowerOn = true;
+    }
+
+    public void powerOff() {
+        this.isPowerOn = false;
+    }
+
+    public void fillGas(double gas) {
+        if (this.gas + gas <= this.capacity)
```

Comment:
And else clause is need that sets the gas to the capacity

Reply:
fixed